txp/Pos/Txp/Logic/Local.hs

```diff
@@ -62,31 +67,39 @@ txProcessTransaction itw@(txId, txAux) = do
                    toList $
                    NE.zipWith (liftM2 (,) . Just) _txInputs resolvedOuts
     pRes <- modifyTxpLocalData $
-            processTxDo resolved toilEnv tipDB itw
+            processTxDo resolved toilEnv tipDB itw bootEra
     case pRes of
         Left er -> do
             logDebug $ sformat ("Transaction processing failed: "%build) txId
             throwError er
         Right _   ->
             logDebug (sformat ("Transaction is processed successfully: "%build) txId)
   where
+    notBootRelated =
+        let txDistr = getTxDistribution $ taDistribution txAux
+            inBoot (s,_) = s `HS.member` genesisBootProdStakeholders
+        in NE.filter (\txOutDistr -> any (not . inBoot) txOutDistr) txDistr
```

Comment:
Don't we want to also check that distribution corresponds to `genesisSplitBoot`? I. e. that we can't distribute everything to one stakeholder. Or that we can't throw away stake.